log4j-layout-jackson/src/main/java/org/apache/logging/log4j/jackson/AbstractJacksonLayout.java

```diff
@@ -216,11 +222,126 @@ public Object getLogEvent() {
     protected static final String DEFAULT_EOL = "\r\n";
     protected static final String COMPACT_EOL = Strings.EMPTY;
     private static LogEvent convertMutableToLog4jEvent(final LogEvent event) {
-        // TODO Jackson-based layouts have certain filters set up for Log4jLogEvent.
-        // TODO Need to set up the same filters for MutableLogEvent but don't know how...
-        // This is a workaround.
```

Comment:
These comments do not appear to be correct. Rhe code is confusing because we register a Jackson property filter using id `org.apache.logging.log4j.core.impl.Log4jLogEvent`, however the code is not at all specific to Log4jLogEvent.